setup.cfg

```diff
@@ -0,0 +1,46 @@
+[metadata]
+name = qutip-qip
+url = https://github.com/qutip/qutip-qip
+description = The QuTiP quantum information processing package
+long_description = file: README.md
+long_description_content_type = text/markdown; variant=GFM
+keywords = quantum, physics, dynamics
+license = BSD 3-Clause License
+license_files = LICENSE
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering
+    Operating System :: MacOS
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: Microsoft :: Windows
+author = Alexander Pitchford, Paul D. Nation, Robert J. Johansson, Chris Granade, Arne Grimsmo, Nathan Shammah, Shahnawaz Ahmed, Neill Lambert, Eric Giguere, Boxi Li, Jake Lishman
+author_email = qutip-admin@googlegroups.com
+platforms = Linux, Mac OSX, Unix, Windows
+version = 0.1.0
+
+[options]
+package_dir=
+    =src
+packages = find:
+zip_safe = False
+include_package_data = True
+install_requires =
+    numpy>=1.16.6
+    scipy>=1.0
+    qutip>=4.6
```

Comment:
Since you use `QobjEvo`'s `tlist`, you're probably not compatible with QuTiP v5 right now - you can be `qutip>=4.6,<5`.

Reply:
Ok, I'll add that. Although there is no qutip v5 at the moment :)

Reply:
Yeah, though this version of your package will still be available on `pip` after it is.  We should probably pin `QuTiP` to the current major releases of Numpy and SciPy as well.